digdag-cli/src/main/java/io/digdag/cli/Main.java

```diff
@@ -133,6 +134,7 @@ protected void configure()
         jc.addCommand("upload", injector.getInstance(Upload.class));
         jc.addCommand("download", injector.getInstance(Download.class));
 
+        jc.addCommand("projects", injector.getInstance(ShowProjects.class), "projects");
```

Comment:
Third argument is alias. You can set alias same as workflow -> workflows or as secrets -> secret.